athena-federation-integ-test/README.md

```diff
@@ -88,7 +88,35 @@ Provide implementation for the following 4 abstract methods in the test class:
      * access to multiple connector-specific AWS services (e.g. DynamoDB, Elasticsearch etc...)
      * @return A policy document object.
      */
-    protected abstract PolicyDocument getConnectorAccessPolicy();
+    protected abstract Optional<PolicyDocument> getConnectorAccessPolicy();
```

Comment:
public API changes should not be taken lightly. Why does this need to change?

Reply:
When I first worked on the integ-test fwk, I based it on the DynamoDb connector which has a dynamodb-specific policy. Later, when working on the Redshift integ-tests, I realized that the connector-specific policy is optional (e.g. Redshift doesn't need one). Rather than returning null and then checking for nulls in the framework, I figured that using Optional<> would be preferable here.